x-pack/plugins/observability_ai_assistant/server/service/client/index.ts

```diff
@@ -502,6 +504,23 @@ export class ObservabilityAIAssistantClient {
         params: subAction,
       });
 
+      const responseBodyChunks = await getResponseBodyChunksFromStream(
+        // executeResult.data as Readable,
+        (executeResult.data as Readable).pipe(new PassThrough()),
+        this.dependencies.logger
+      );
+
+      try {
+        const tokenCount = await getTokenCountFromOpenAI({
+          requestBody: subAction.subActionParams.body,
+          responseBodyChunks,
+        });
+
+        console.log({ tokenCount });
+      } catch (e) {
+        console.log('could not get tokens');
+      }
+
```

Comment:
This still doesn't work (same problem as in https://github.com/elastic/kibana/pull/177373)